tasks/libs/ciproviders/github_actions_tools.py

```diff
@@ -14,6 +14,45 @@
 from tasks.libs.common.git import get_default_branch
 
 
+def trigger_buildenv_workflow(workflow_name="runner-bump.yml", github_action_ref="master", new_version=None):
+    """
+    Trigger a workflow to bump windows gitlab runner
+    """
+    inputs = {}
+    if new_version is not None:
+        inputs["new-version"] = new_version
+
+    print(
+        "Creating workflow on buildenv on commit {} with args:\n{}".format(  # noqa: FS002
+            github_action_ref, "\n".join([f"  - {k}: {inputs[k]}" for k in inputs])
+        )
+    )
+
+    # Hack: get current time to only fetch workflows that started after now
+    now = datetime.utcnow()
+
+    gh = GithubAPI('DataDog/buildenv')
+    result = gh.trigger_workflow(workflow_name, github_action_ref, inputs)
+
+    if not result:
+        print(f"Couldn't trigger workfglow run. result={result}")
+        raise Exit(code=1)
```

Comment:
Attempting to exit the program from within a "library" function looks off to me.

Reply:
I was mimicking our currents macOS builds, I kinda agree but since the function by itself is already very specific I'd say it's fine wdyt ?

Reply:
I did realize that a big part of this function was extracted from the one for triggering macos builds. Up to you.

Reply:
I'll keep it that way for now